docs/standalone.md

```diff
@@ -3,73 +3,79 @@ Runnig Heapster standalone on a host with cAdvisor
+The build steps are as follows, [go](https://github.com/golang) and [godep](https://github.com/tools/godep) need be installed in system first.  
 
 
 **Step 1: Clone the heapster github repo**
 
-``` 
+```shell 
 $ git clone https://github.com/kubernetes/heapster.git
 ```
 
 **Step 2: Build the source code**
 
-``` 
+```shell 
 $ cd kubernetes/heapster
 $ make
 ```
 
 Then you can get the heapster binary in current path.
 
-### Running a cAdvisor
-cAdvisor can run in a docker or standalone outside of Docker to monitor the whole machine. This guide run a cAdvisor standalone out of Docker.cAdvisor is a static Go binary with no external dependencies. To run it standalone all you should need to do is run it! Note that some data sources may require root priviledges.You can get cAdvisor binary from the [Release Page](https://github.com/google/cadvisor/releases)
+### Run a cAdvisor
+cAdvisor can run in a docker or standalone outside of Docker to monitor the whole machine. This guide run a cAdvisor standalone outside of Docker.  
+cAdvisor is a static Go binary with no external dependency.Note that some data source may require root priviledges.  
+You can get cAdvisor binary from the [Release Page](https://github.com/google/cadvisor/releases)  
 
-``` 
+```shell
 $ ./cadvisor  
 ```
 
 cAdvisor is now running (in the foreground) on http://localhost:8080/.
 
-## Starting a heapster
-
-### Config Cadvisor source
-Heapster support two types of Cadvisor source: standalone & CoreOS; Doc of source configuration is [HERE](https://github.com/kubernetes/heapster/blob/master/docs/source-configuration.md)
+## Start a heapster
 
-External cadvisor source "discovers" hosts from the specified file. Use it like this:
+### Config cAdvisor source
+Heapster supports two types of cAdvisor source: `standalone` & `CoreOS`;   
+Doc of source configuration is [HERE](https://github.com/kubernetes/heapster/blob/master/docs/source-configuration.md)
+External cAdvisor source "discovers" hosts from the specified file. Use it like this:
 
-```
+```shell
 --source=cadvisor:external[?<OPTIONS>]
 ```
 
 The following options are available:
 
 * `standalone` - only use `localhost` (default: `false`)
-* `hostsFile` - file containing list of hosts to gather cadvisor metrics from (default: `/var/run/heapster/hosts`)
-* `cadvisorPort` - cadvisor port to use (default: `8080`)
+* `hostsFile` - file containing list of hosts to gather cAdvisor metrics from (default: `/var/run/heapster/hosts`)
+* `cadvisorPort` - cAdvisor port to use (default: `8080`)
 
 Here is an example:
+
 ```shell
 $ ./heapster --source="cadvisor:external?cadvisorPort=4194"
 ```
 
-Because heapster needs a specified file to config external cadvisor source, we need to provide one.
-The `hostsFile` parameter defines a list of hosts to poll for metrics and must be in JSON format. See below for an example:
+Because heapster needs a specified file to config external cAdvisor source, we need to provide one.  
```

Comment:
Hi, @vishh , adding two white space at the end, aims to place the sentence in a new line : )